server/src/main/java/org/opensearch/gateway/remote/RemoteClusterStateService.java

```diff
@@ -795,7 +795,9 @@ private ClusterMetadataManifest uploadManifest(
                 uploadedCoordinationMetadata,
                 uploadedSettingsMetadata,
                 uploadedTemplatesMetadata,
-                uploadedCustomMetadataMap
+                uploadedCustomMetadataMap,
+                clusterState.routingTable().version(),
+                new ArrayList<>()
```

Comment:
Will this be passed with actual list in the subsequent PR?

Reply:
Yes, this is handled in the PR for write flow: https://github.com/opensearch-project/OpenSearch/pull/13870/files#diff-1a437aed1331a557c8bdefb7b37e465d2da1670a059a9aafa1fcb307310cb315R859